extbase-best-practices/sql-tca.tex

```diff
@@ -10,7 +10,7 @@ \section{Remove unneeded default-generated columns}
 \begin{description}
   \item[hidden, starttime, endtime] if you do not need this restrictions
   \item[sys\_language\_uid, l10n\_parent, l10n\_diffsource, KEY language] if the records in this table will never be translated
-  \item[t3ver\_*] if you do not need versioning or workspaces for this table
+  \item[t3ver\_*] if you do not need versioning or workspaces for this table. Also set \texttt'versioningWS'=>false,, in \textttTCA/tx_extension_domain_model_table.php
```

Comment:
The correct syntax for texttt would be with curly braces:

\texttt{foo ... bar}

This also allows you to have spaces in the monospace parts.

Reply:
I changed it.